Add index.html and styles.css

Arch-Noize opened the pull request:
Following the "mobile first" design and [template #3](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=48%3A1589&t=dEJVCqc111mjPdDy-1) from Figma, this pull includes:
- Responsive design, stylized to match a template
- Flexbox implementation

Arch-Noize committed:
Delete Murple logo

It is copied from the template and will not be used

Arch-Noize committed:
Delete app screenshot

It is copied from the template and will not be used.

Arch-Noize committed:
Update README.md

Edit the template to better fit the project's features.

Arch-Noize committed:
Add .gitignore

Arch-Noize committed:
Add linters.yml

Arch-Noize committed:
Add index.html

Arch-Noize committed:
Add styles.css

Arch-Noize committed:
Update index.html

Added the text from the template

Arch-Noize committed:
Add Images folder and content

The Images folder contains website content for background images and designs

Arch-Noize committed:
Update index.html

Added header, cleared text, and uses of content from the Images folder

Arch-Noize committed:
Update styles.css

Now includes all site styling and implemented flexbox designing

Arch-Noize committed:
Update styles.css

Used to have borders outlining the flexboxes, they have been removed

Arch-Noize committed:
Remove inline css from index.html

Arch-Noize committed:
Update styles.css

Arch-Noize committed:
Fix linters error

Stylelint picked up several css selectors not having a space before '{'

Arch-Noize committed:
Fix Superlint errors

Arch-Noize committed:
Fix Superlint errors

Arch-Noize committed:
Fix Superlint errors

souad988 commented on index.html:
```diff
@@ -0,0 +1,40 @@
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="bar">
+        <img src="/Images/app_bar.png"/>
+        <img src="/Images/Light.png"/ class="top";>
+    </header>
```
Very well done by using header to wrap your header content 👍🏼  , but the first image should not be added as it is just to show you are in mobile the header includes only a humberger icon aligned to left and it should be wraped in a nav tag for semantic reasons

Arch-Noize replied:
so the image of the hamburger should be in a nav tag then?

souad988 commented on index.html:
```diff
@@ -0,0 +1,40 @@
+        <img src="/Images/app_bar.png"/>
+        <img src="/Images/Light.png"/ class="top";>
+    </header>
+    <div class="container">
+        <section class="head">
```
good here as well by trying to use section tags but it is meant to wrap the hall headlines content including social media part as all these item are sharing the same purpus as to introduce your pofile also to wrap all your project sections as you will be adding other section in the coming weeks you need to wrap in a main section only the header and footer would be excluded

Arch-Noize replied:
Could you.... please rephrase that? I don't understand what you are trying to say.

souad988 commented on index.html:
```diff
@@ -0,0 +1,40 @@
+        <section class="head">
+            <h2>
+                Hey there. I’m Santiago. 
+                <section class="title">
+                    I’m a software developer
+                </section>   
```
also here please remove this section tag and use hx as those are titles

Arch-Noize replied:
it is using the HX tag, its simply put as a separate section so it can be stylized. If I were to write something like
`<h2 class="title"> I’m a software developer </h2>` it would lose the CSS styling. Can I use div maybe?

souad988 commented on index.html:
```diff
@@ -0,0 +1,40 @@
+            </p>
+        </section>
+
+        <div class="socials">
+            <img src="/Images/social.png"/>
+        </div>
```
social media bloc is not one image but a list of icons each of them is wraped in anchor a tag with a valid href likn to your social accounts and as it is a list of links please use ul li tags

Arch-Noize replied:
How can I use ul and li tags on images?

souad988 reviewed (changes requested):
Hi @Arch-Noize , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
Good styling
Good use of github workflows
Use of microverse readme template

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Arch-Noize committed:
Update index.html

Fixed social icons to be individual anchor tags that lead to my social network and made menu a nav bar

Arch-Noize committed:
Update styles.css

Now properly shows each social icon as a separate item requiring special spacing

TuyishimireEric commented on index.html:
```diff
@@ -0,0 +1,42 @@
+    </header>
+    <div class="container">
+        <section class="head">
+            <h2> 
```
- [ ] **Kindly as suggested by the previous reviewer, 👆 👆 ** the <header> element represents the topmost section of the page, and the <section> elements are used to divide the content into meaningful sections. By utilizing these elements properly. 👍 

   - I think it would be better for the container of this section to use a `header` tag. 👌😎
    Typically the header tag is used as a page header, it contains the site's **title** or **logo** (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. 👍 

example:
```
        <header class="navbar-left">
            <a class="logo-text">My Logo</a>
            <nav>
                <img src="./icons/Icon - Menu.png" alt="burguer menu">
             </nav>
        </header>

Arch-Noize replied:
sorry your reply seems to be cut at the start. the container class I have holds the text too so... Should I make the container  class be for the header and pull all the text inside too, or should I make a new container in header that includes the logo, and turn it into header? 

Something like
```
    <header class="bar-container">
         <nav>
            <img src="/Images/Light.png"/>
         </nav>
        <img src="/Images/app_bar.png"/>
    </header>
    <div class="container">
        <section class="head">
            <h2> 
                Hey there. I’m Santiago. 
                <div class="title"> I’m a software developer </div>
            </h2>
```

or something like

```
    <nav>
        <img src="/Images/Light.png"/>
    </nav>
    <header class="container">
        <img src="/Images/app_bar.png"/>
        <section class="head">
            <h2> 
                Hey there. I’m Santiago. 
                <div class="title"> I’m a software developer </div>
            </h2>
```

TuyishimireEric replied:
You may take the first option of a header as a container

TuyishimireEric commented on index.html:
```diff
@@ -0,0 +1,42 @@
+            <a href="https://medium.com/"> <img src="Images/medium.png" alt="Medium"/> </a>
+            <a href="https://github.com/Arch-Noize"> <img src="Images/github.png" alt="GitHub"/> </a>
+            <a href="https://www.angellist.com/"> <img src="Images/angellist.png" alt="Angellist"/> </a>
+        </div>
```
- [ ] I think it would be better to use `ul` tag to group, and `li` to list eash icon As suggested by the previous reviewer too 👍.

   The `<ul>` tag in HTML is used to create bulleted lists. It provides semantic structure, improves readability and organization, enhances accessibility, allows for customization through CSS, and promotes maintainability by separating content structure from presentation. In summary, the `<ul>` tag is valuable for creating unordered lists with clear visual cues and meaningful structure. 👍 👍 

- [ ] **[Optional]** It would be better also to adjust the links of the icons as they are not visible on my side, Kindly check this for the best user experience 👍

Arch-Noize replied:
I don't mind using the ul tag, but it would make a bulleted list when the template in question doesn't uses it and the activity doesn't require it.

I can perfectly add it but I just want to be on record that it will differ from the template's design 

Also, odd that the images don't load for you. Will see what's going on there

TuyishimireEric replied:
@Arch-Noize, It is possible to remove the bullets in CSS by adding this line `list-style-type: none ;` This will remove the bullets 😎

TuyishimireEric reviewed (changes requested):
# Status: REQUIRED CHANGES :recycle: 
### Hi  @Arch-Noize  👋 👋 
#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
There are some issues that you still need to work on to go to the next project but you are almost there!
![](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)
## Highlight 
✔ Used the correct Git Flow 👍 
✔ No Linter Errors👍 
✔ Great communicator👍 
## Required Changes ♻️

- [ ]  Check the comments below 

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Arch-Noize committed:
Update index.html

Moved nav bar into a header and added ul tag for social links

Arch-Noize committed:
Update styles.css

Modify ul tag to not have dots

Arch-Noize committed:
Fix navbar in index.html

Arch-Noize committed:
Update styles.css

Added new nav bar styling

Arch-Noize committed:
Deleted comment index.html

Arch-Noize committed:
Fix Superlinter error

victorokoroji commented on index.html:
```diff
@@ -0,0 +1,44 @@
+            </p>
+        </section>
+        <div class="socials">
+            <ul>
+                <li> <a href="https://twitter.com/NoizeSalt"> <img src="Images/twitter.png" alt="Twitter"/> </a> </li>
+                <li> <a href="https://www.linkedin.com/in/santiago-ruido-a1404880/"> <img src="Images/Linkedin icon.png" alt="LinkedIn"/> </a> <li>
+                <li> <a href="https://medium.com/"> <img src="Images/medium.png" alt="Medium"/> </a> </li>
+                <li> <a href="https://github.com/Arch-Noize"> <img src="Images/github.png" alt="GitHub"/> </a> </li>
+                <li> <a href="https://www.angellist.com/"> <img src="Images/angellist.png" alt="Angellist"/> </a> </li>
+             </ul>
```
[OPTIONAL]

- Your social media icons are currently not displaying. Kindly ensure to use the correct image paths so they can be visible on the UI
![image](https://github.com/Arch-Noize/setup-and-mobile-first/assets/63473584/f8683740-df9e-4803-a0a2-44c0e49761c1)

victorokoroji commented on index.html:
```diff
@@ -0,0 +1,44 @@
+</head>
+<body>
+    <header class="bar-container">
+        <nav>
+           <img src="/Images/Light.png"/>
+           <img src="/Images/app_bar.png"/>
+        </nav>
```
[OPTIONAL]

- Kindly remove this part from your webpage as they are added to the design to just represent that you are on the mobile view screen

![image](https://github.com/Arch-Noize/setup-and-mobile-first/assets/63473584/a43d77aa-f238-4505-86f2-c7e42736d0d2)

victorokoroji commented on index.html:
```diff
@@ -0,0 +1,44 @@
+   </header>
+   <div class="container">
+       <section class="head">
+           <h2> 
+               Hey there. I’m Santiago. 
+               <div class="title"> I’m a software developer </div>
+           </h2>
```
[OPTIONAL]

- As this will be the main heading of your website, I will recommend that you use an `h1` tag instead of` h2 `tag so that screen readers and SEO will will be able to understand and index your page properly, hence improving your website accessibility

victorokoroji reviewed (approved):
Hi @Arch-Noize

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested change implemented ✅ 

<img src="https://media3.giphy.com/media/xT0xezQGU5xCDJuCPe/giphy.gif"/>

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Arch-Noize committed:
Add images for social icons

Arch-Noize committed:
Remove social.png

No longer needed, as there's images of each icon alone

Arch-Noize committed:
Update index.html

Changed h1 tag to h2 at start, removed phone status bar image

Arch-Noize committed:
Update styles.css

Fixed styling and cleared unnecessary repetition
